test/runner/utils.ts

```diff
@@ -1,8 +1,8 @@
 import { readFile, statSync, readdirSync, writeFile, existsSync } from 'fs';
 
-const PrettyError = require('pretty-error');
+import PrettyError from 'pretty-error';
 
-export const snapshotsFolderName = '__snapshots__'
+export const snapshotsFolderName = '__snapshot__'
```

Comment:
it's more than 1 snapshot in that folder so should add (s) behind

Reply:
you need to rename the folder if you put it that way. ;)

will do it later.